pkgs/development/tools/rust/cargo-generate/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, rustPlatform, Security, openssl, pkgconfig, libiconv, curl }:
+
+rustPlatform.buildRustPackage rec {
+  name = "cargo-generate-${version}";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "ashleygwilliams";
+    repo = "cargo-generate";
+    rev = "v${version}";
+    sha256 = "0n6na6xq4bvs9hc7vc86qqmlrkv824qdmja27b21l2wz3l77r4jb";
+  };
+
+  cargoSha256 = "00fgzh1s63rr1vs3ahra604m81fc4imx3s09brw2y0n46syhwypi";
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildInputs = [ openssl libiconv curl ]
```

Comment:
Are we sure `libiconv` and `curl` will be need on linux?

Reply:
yes indeed. Sorry I just wanted to push it from mac so I can refactor it on my linux machine.